include/glow/Quantization/Base/Base.h

```diff
@@ -53,7 +53,7 @@ struct QuantizationTransform32To8 {
   int32_t transform(int32_t input) {
     // The operation x >> y is rounded down to negative infinity. To get to
     // round-nearest we add (1 << (shift - 1)) to the value prior to shifting.
-    int rtn = (1 << (post - 1));
+    int rtn = (post > 0) ? (1 << (post - 1)) : 0;
```

Comment:
Please update the comments since "rtn" is updated.

Reply:
Done.